app/serializers/serialize_mentor_session_request.rb

```diff
@@ -10,6 +10,7 @@ def call
       uuid: request.uuid,
       comment: SerializeMentorDiscussionPost.(Mentor::RequestComment.from(request), user),
       is_locked: request.locked?,
+      status: request.status,
```

Comment:
```suggestion
 
```

Reply:
I've added this, but didn't end up using it. Do you think we might need it? Is there a chance mentors could accidentally open a cancelled request?